assets/src/domain/types.ts

```diff
@@ -0,0 +1,114 @@
+import type { EventEditorData } from '@edtrInterfaces/types';
+import type { WpPluginsPageData } from './wpPluginsPage/types';
+import type {
+	CurrencyProps,
+	CurrentUserProps,
+	GeneralSettings,
+	LocaleProps,
+	SiteUrlProps,
+} from '@application/valueObjects/config/types';
+
+/**
+ * Contains the API related information
+ */
+export interface ApiData {
+	nonce?: string;
+	restBaseUrl?: string;
+	graphqlEndpoint?: string;
+}
+
+/**
+ * This can contain asset information like
+ * - base URL to assets directory
+ * - URLs to specific images like brand images etc.
+ */
+export interface AssetsData {
+	url: string; // URL to assets directory
+}
+
+export interface ConfigData {
+	// Better to name the type as "CurrencyConfig" instead of "CurrencyProps"
+	currencyConfig?: CurrencyProps;
```

Comment:
or even just `currency` since it is already part of the `ConfigData` object ?

Reply:
Yes, better 👍